lib/interpreters/compute_several_solutions.rb

```diff
@@ -238,37 +238,47 @@ def self.find_best_heuristic(service_vrp)
     end
 
     def self.select_best_heuristic(vrp)
-      vehicles = vrp[:vehicles] || []
-      services = vrp[:services] || []
-      shipments = vrp[:shipments] || []
+      vehicles = vrp.vehicles
+      services = vrp.services
 
       loop_route = vehicles.any?{ |vehicle|
-        if (vehicle.start_point_id.nil? || vehicle.end_point_id.nil?) && (vehicle.start_point.nil? || vehicle.end_point.nil?)
+        if (vehicle.start_point_id.nil? || vehicle.end_point_id.nil?) &&
```

Comment:
Regarding what we discussed this morning, I guess we can simply use one oh these lines ?